seed/index.js

```diff
@@ -0,0 +1,55 @@
+// const House = require('../db/Schemas/Listing.js');
+
+
+//loop 100 times
+const seed = () => {
+    const houses = [];
+    for (let i = 0; i < 100; i++) {
+        const house = {};
+        let pad = i;
+
+        if (pad.toString().length < 2) {
+            house['Listing_id'] = '00' + pad.toString();
+        } else {
+            house['Listing_id'] = '0' + pad.toString();
+        }
+
+        house['Price'] = Math.floor(Math.random() * 5999999) + 100000;
```

Comment:
This is a nice and simple way to generate pricing, which is essentially all my db is.  I'm trying to use Faker.js just to practice using it, were you not interested in using their 'cards'?

Reply:
Forgot about Faker completely, I had an idea and ran with it. Didn't take much